episodes/tabular-data-collection.md

```diff
@@ -27,59 +27,203 @@ After following this episode, learners will be able to:
+
+In the MET museum dataset Alex is using, each row is an object (like a painting or sculpture), and each cell is a data point.
+
+Alex’s dataset looks like a spreadsheet, but underneath, each column contains a specific **data type**. Knowing these helps to avoid errors and choose the right tools for analysis.
+
+#### Basic data types
+
+Before we look at different types of variables, here are some common data types you'll encounter:
+
+- **String**: Text or characters, like `"Claude Monet"` or `"Oil on canvas"`
+- **Integer**: Whole numbers, like `1985`, `42`, or `0`
+- **Float**: Decimal numbers, like `27.5` or `3.14`
+- **Boolean**: True/False values, like `TRUE`, `FALSE`, `Yes`, `No`
+- **Datetime**: Calendar dates or timestamps, like `"2020-01-01"` or `"12/11/2027"`
+
+### What is a variable?
+
+A variable is a characteristic or attribute that can take on different values. In tabular data, variables are usually represented as columns, where each row contains an observation or entry.
+
+However, the concept of a variable is independent of format, it’s not defined by being a column, but by being a consistent type of information collected across observations.
+
+For example, in Alex's MET dataset, variables might include objectid, artistname, or dateacquired.
+
+### Types of variables
+
+#### Numeric variables
+
+Variables that represent measurable quantities. These can be integers or floats. Numeric variables can be _Discrete_, which means they take on specific, separate values (often counts), or _Continuous_, which can take on any value within a range (often measurements).
+
+**Examples**:
+
+- `objectid` → `12345` _(integer, discrete — a unique ID number)_
+- `heightcm` → `23.5` _(float, continuous — a measurement in centimeters)_
+- `objectdate` → `1890` _(integer, discrete — a specific year)_
+
+#### String variables
+
+Free-form or descriptive text.
+
+**Examples**:
+
+- artistdisplayname → `"Claude Monet"` _(string)_
+- title → `"Woman with a Parasol"` _(string)_
+
+#### Categorical variables
+
+Variables that represent groups or categories. These could be strings, integers, or floats - anything used to label a category!  
+Categorical variables can be _Nominal_, which means there is no inherent order (e.g., `artistnationality`), or _Ordinal_, which means the categories follow a logical order (e.g., `popularity`).
+
+**Examples**:
+
+- `gender` → `"Female"`, `"Male"` _(string, nominal)_
+- `medium` → `"Marble"`, `"Bronze"`, `"Oil on canvas"` _(string, nominal)_
+- `istimelinework` → `"Yes"` / `"No"` _(string, nominal — or Boolean: `TRUE` / `FALSE`)_
+- `artistdecade` → `1950`, `1960`, `1980` _(integer, ordinal — ordered decades)_
+
+#### Date/time variables
+
+Variables that represent dates or times.
+
+**Examples**:
+
+- lastconserv → `"2001-05-12"` _(datetime or string)_
+- objectdate → `1990` _(integer)_, `"ca. 1890"` _(string)_
+
+::::::::::::::::::::::caution
```

Comment:
```suggestion
::::::::::::::::::::::callout
```
Maybe use a callout for this one? It's providing further context, rather than indicating something that could lead to errors / issues in analysis.